lib/active_record_doctor/detectors/unindexed_foreign_keys.rb

```diff
@@ -27,7 +27,7 @@ def message(table:, columns:)
       def detect
         each_table(except: config(:ignore_tables)) do |table|
           each_column(table, except: config(:ignore_columns)) do |column|
-            next unless named_like_foreign_key?(column) || foreign_key?(table, column)
```

Comment:
Is `named_like_foreign_key?` needed or has it become unused?

Reply:
forgot to remove. Removed now.